ocs_ci/utility/utils.py

```diff
@@ -2074,14 +2074,18 @@ def set_selinux_permissions(workers=None):
     cmd = ['/usr/sbin/setsebool -P container_use_cephfs on']
     cmd_list = cmd.copy()
     if not workers:
-        workers = get_typed_worker_nodes(os_id="rhel")
-    for worker in workers:
+        worker_nodes = get_typed_worker_nodes(os_id="rhel")
```

Comment:
do we need those changes or you find it more readable like this?

Reply:
I modified the function to have specific worker nodes. Else function would take all rhel nodes for applying permission, post cluster setup